src/defaultOptions.js

```diff
@@ -29,6 +29,10 @@ const everyMonthOptions = {
         amount: '100',
         bgColor: '#A0CBFE',
       },
+      {
+        amount: '500',
```

Comment:
```suggestion
        amount: '200',
```
I like 200/month as the largest, 500/month feels a bit much ...